sdk/eventhub/azure-eventhubs/README.md

```diff
@@ -200,59 +182,32 @@ with client:
 
 ### Async publish events to an Event Hub
```

Comment:
"Publish events to an Event Hub asynchronously"

Reply:
Updated